test/stack-test-inner-loop.sh

```diff
@@ -33,8 +33,30 @@ while ! odo log | grep -q "CWWKF0011I: The defaultServer server"; do
     fi
 done
 
+echo -e "\n> Run odo test"
+odoTestResults=$(odo test)
```

Comment:
Wouldn't you usually test $? rather than parsing the output ?

Reply:
Yup - good point.